dev/release/rat_exclude_files.txt

```diff
@@ -7,7 +7,9 @@
 *.csv
 *.json
 *.snap
-.github/ISSUE_TEMPLATE/question.md
+.github/ISSUE_TEMPLATE/bug_report.yaml
```

Comment:
Why exclude these? The license header can be added to issue templates without a problem in my experience.

Reply:
"Prior art" :) This was how [I found question.md suppressed lint failures](https://github.com/apache/arrow/commit/55b9263b8106e8678a1f03a7ba98f4de6f56fd51), so I followed that pattern here.